CommonLibSF/include/RE/P/PlayerCharacter.h

```diff
@@ -361,7 +361,7 @@ namespace RE
 		std::uint64_t unk0F38;        // 0F38
 		std::uint64_t unk0F40;        // 0F40
 		std::uint64_t unk0F48;        // 0F48
-		std::uint64_t unk0F50;        // 0F50
+		TESObjectREFR* crosshairRef;  // 0F50
```

Comment:
You should at least add a comment mentioning that it could be a smart pointer if you haven't confirmed it.